case/src/test/kotlin/com/ritense/case/service/CaseDefinitionServiceTest.kt

```diff
@@ -207,36 +217,152 @@ class CaseDefinitionServiceTest {
     }
 
     @Test
-    fun `should fail to validate list column dto`() {
+    fun `should fail to validate list column dto on create`() {
         val caseDefinitionName = "aName"
-        val listColumnDto = getListColumnDto(
+        val listColumnDto = getListColumnDtoToFirstName(
             DisplayType("enum", EnumDisplayTypeParameter(emptyMap()))
         )
         whenever(documentDefinitionService.findIdByName(caseDefinitionName))
             .thenReturn(JsonSchemaDocumentDefinitionId.newId("aName"))
-        whenever(caseDefinitionListColumnRepository.findByIdCaseDefinitionName(caseDefinitionName))
+        whenever(
+            caseDefinitionListColumnRepository.findByIdCaseDefinitionNameOrderByOrderAscSortableAsc(
+                caseDefinitionName
+            )
+        )
             .thenReturn(
                 emptyList()
             )
         doNothing().whenever(documentDefinitionService).validateJsonPath(caseDefinitionName, listColumnDto.path)
         val exception = assertThrows<InvalidListColumnException> {
-            service.createListColumn(caseDefinitionName, listColumnDto)
+            service.upsertListColumn(caseDefinitionName, listOf(listColumnDto), Operation.CREATE)
         }
         verify(documentDefinitionService).findIdByName(caseDefinitionName)
-        verify(caseDefinitionListColumnRepository).findByIdCaseDefinitionName(caseDefinitionName)
+        verify(caseDefinitionListColumnRepository).findByIdCaseDefinitionNameOrderByOrderAscSortableAsc(
+            caseDefinitionName
+        )
         verify(documentDefinitionService).validateJsonPath(caseDefinitionName, listColumnDto.path)
         assertEquals("Display type parameters are invalid for type enum.", exception.message)
 
     }
 
-    private fun getListColumnDto(displayType: DisplayType): CaseListColumnDto {
+    private fun getListColumnDtoToFirstName(displayType: DisplayType): CaseListColumnDto {
         return CaseListColumnDto(
             title = "First name",
             key = "first-name",
             path = "doc:firstName",
             displayType = displayType,
             sortable = true,
-            defaultSort = ColumnDefaultSort.ASC
+            defaultSort = ColumnDefaultSort.ASC,
+            order = 1
+        )
+    }
+
+    private fun getListColumnDtoLastName(displayType: DisplayType): CaseListColumnDto {
```

Comment:
We usually put private methods on the bottom of the file. Below all @.Tests and all other public methods